rpc/net.go

```diff
@@ -168,7 +168,7 @@ func (s *NetService) takeOffer(who peer.ID, offerID types.Hash, providesAmount *
 		return nil, fmt.Errorf("failed to initiate protocol: %w", err)
 	}
 
-	skm := swapState.SendKeysMessage()
+	skm := swapState.SendKeysMessage().(*message.SendKeysMessage)
```

Comment:
```suggestion
	skm, ok := swapState.SendKeysMessage().(*message.SendKeysMessage)
	if !ok {
	    panic("expected SendKeysMessage")
	}
```
this shouldn't happen, but good to check again

Reply:
We'll get a panic with a line number in both cases. Is a slightly prettier panic worth additional code complexity?